config/newrelic.yml

```
@@ -1,4 +1,9 @@
+common: &default_settings
+  application_logging:
+    forwarding:
+      enabled: false
 production:
+  <<: *default_settings
   agent_enabled: true
```

If production is our only configured environment, do we have need for the common block?

```suggestion
production:
  agent_enabled: true
  application_logging:
    forwarding:
      enabled: false
```

---

We don't strictly need it, no.  My goal was to communicate the intention that we always want it disabled in all environments (though maybe a comment is a better path to do that).

---

Fair 🙂 

I've moved the config into `production` and added a comment to keep it consistent.